internal/sgbouncewizard/sgbouncewizard.go

```diff
@@ -577,6 +570,50 @@ func (srv *Server) Serve(Port int) {
 		})
 	})
 
+	r.Route("/ws", func(r chi.Router) {
+		r.Get("/", func(w http.ResponseWriter, r *http.Request) {
+			m.HandleRequest(w, r)
+		})
+	})
+
+	m.HandleMessage(func(s *melody.Session, msg []byte) {
+		log.Println(string(msg))
+		params := strings.Split(string(msg), " ")
+		lock.Lock()
+		if params[0] == "edit" {
```

Comment:
More of a stylistic choice but I'd consider using a `switch` vs `if/else if` here